.gitignore

```diff
@@ -19,7 +19,7 @@ yarn-error.log*
 .node_repl_history
 
 # Output of 'npm pack'
-*.tgz
+#*.tgz
```

Comment:
^^ this was changed for committing local / temporary pnpm packages only (and could be removed once rdy for merge w/ released pkgs)